### PR TITLE
Add linux-aarch64 CI build

### LINKED ISSUE #46: linux aarch64 release
Can we please include an `aarch64` release as babashka supports this architecture?

```Dockerfile
FROM arm64v8/ubuntu

RUN apt update
RUN apt install -y curl

RUN curl -sLO https://raw.githubusercontent.com/babashka/babashka/master/install && \
    chmod +x install && \
    ./install

RUN bb "(babashka.pods/load-pod 'org.babashka/aws \"0.1.0\")"
```

Test build by running:
```bash
docker build -t aws-arm .
```

Error on missing executable for arch:
```
#9 0.394 ----- Error --------------------------------------------------------------------
#9 0.394 Type:     java.lang.IllegalArgumentException
#9 0.394 Message:  No executable found for pod org.babashka/aws (0.1.0) and OS Linux/aarch64
#9 0.394 Location: <expr>:1:1
#9 0.394 
#9 0.394 ----- Context ------------------------------------------------------------------
#9 0.395 1: (babashka.pods/load-pod 'org.babashka/aws "0.1.0")
#9 0.395    ^--- No executable found for pod org.babashka/aws (0.1.0) and OS Linux/aarch64
#9 0.395 
#9 0.395 ----- Stack trace --------------------------------------------------------------
#9 0.395 user - <expr>:1:1
#9 0.395 
------
```

Please let me know if you don't have time to alter the cicd pipelines, I can try to get some time to do this.

Thanks!! 😎

### PR DESCRIPTION
**Pull request by cap10morgan**
Need to test this in CircleCI still

Closes #46

### CODE REVIEW

**Comment by borkdude** [comment]
Can you do another dummy commit? I enabled building PRs in CircleCI which I forgot.

File: .circleci/config.yml [hunk 1]
```diff
@@ -188,6 +188,65 @@ jobs:
...
+    machine:
+      enabled: true
+      image: ubuntu-2004:202101-01
+    resource_class: arm.large
+    working_directory: ~/repo
+    environment:
+      LEIN_ROOT: "true"
+      GRAALVM_HOME: /home/circleci/graalvm-ce-java11-21.3.0
+      BABASHKA_STATIC: "true"
+      BABASHKA_MUSL: "true"
```
Comment (borkdude) [inline .circleci/config.yml:201]: MUSL doesn't work on aarch64 unfortunately.

**Comment by borkdude** [comment]
I updated the compile script on master.  Can you merge with that?

File: .circleci/config.yml [hunk 2]
```diff
@@ -220,7 +220,6 @@ jobs:
           command: |
             sudo apt-get update
             sudo apt-get -y install g++
-            sudo -E script/setup-musl
```
Comment (borkdude) [inline .circleci/config.yml]: Please don't remove this! It's still used by the other linux builds.

Comment (borkdude) [reply .circleci/config.yml]: Oh, wait, my bad. This is in the build for aarch64 right. Sorry!

Comment (cap10morgan) [reply .circleci/config.yml]: It's still there for the amd64 build. I only removed it from the aarch64 build.

File: .circleci/config.yml [hunk 3]
```diff
@@ -188,6 +188,63 @@ jobs:
...
+          name: Install native dev tools
+          command: |
+            sudo apt-get update
+            sudo apt-get -y install g++
+      - run:
+          name: Download GraalVM
+          command: |
+            cd ~
+            if ! [ -d graalvm-ce-java11-21.3.0 ]; then
+              curl -O -sL https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-21.3.0/graalvm-ce-java11-linux-aarch64-21.3.0.tar.gz
```
Comment (borkdude) [inline .circleci/config.yml:228]: Need to install the aarch64 graalvm here.

**Pull request merged**